src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const role = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (
+        <p className="Person__age">{`I am ${age}`}</p>
+      ) : null}
+      {isMarried ? (
+        <p className="Person__partner">{`${partnerName} is my ${role}`}</p>
+      ) : <p className="Person__partner">I am not married</p>}
```

Comment:
```suggestion
      <p className="Person__partner">
        { isMarried
          ? `${partnerName} is my ${role}`
          : `I am not married`}
      </p>
```
try to avoid code dublication